CHANGELOG.md

```diff
@@ -56,15 +75,17 @@ This CHANGELOG follows the format located [here](https://github.com/sensu-plugin
 - Added skel files from Chef Partners cookbook generator (@thomasriley)
 
 ### Changed
+
 - Updated contributing instruction (@majormoses)
 - use `@sensu/chef-cookbooks` for `CODEOWNERS` rather than individual users now that there is a team to refer to instead (@majormoses)
 
 ### Fixed
+
 - moved `CODEOWNERS` into the correct location (@majormoses)
 - updated development dependencies (@majormoses)
 - using a version of `'latest'` for backend and agent providers will now upgrade to the test version
 
-[Unreleased]: https://github.com/sensu/sensu-go-chef/compare/0.1.0...HEAD
+[unreleased]: https://github.com/sensu/sensu-go-chef/compare/0.1.0...HEAD
```

Comment:
I think this might have been an accidental change as I don't see the same being done above

Reply:
Ah, yea. Just a format onsave config with my emacs setup, which uses [prettier](https://prettier.io/). Found the issue and added a local [override](https://prettier.io/docs/en/ignore.html) to avoid it for this section in the future.